opendrift/models/radionuclides.py

```diff
@@ -0,0 +1,620 @@
+        
+        desorption_depth = self.get_config('radionuclide:sediment:desorption_depth')
+        std = self.get_config('radionuclide:sediment:desorption_depth_uncert')
+
+
+        self.elements.z[(sp_out==self.num_lmm) & (sp_in==self.num_srev)] = \
+            self.environment.sea_floor_depth_below_sea_level[(sp_out==self.num_lmm) & (sp_in==self.num_srev)] + desorption_depth
+        if std > 0:
+            logging.debug('Adding uncertainty for desorption from sediments: %s m' % std)
+            self.elements.z[(sp_out==self.num_lmm) & (sp_in==self.num_srev)] += np.random.normal(
+                    0, std, sum((sp_out==self.num_lmm) & (sp_in==self.num_srev)))
+
+
+
+
+
+    def update_radionuclide_diameter(self,sp_in=None,sp_out=None):
+        '''Update the diameter of the radionuclides when specie is changed'''
+        
+        
+        dia_part=self.get_config('radionuclide:particle_diameter')
+        dia_diss=self.get_config('radionuclide:dissolved_diameter')
+        
+        
+        # Transfer to reversible particles
+        self.elements.diameter[(sp_out==self.num_prev) & (sp_in!=self.num_prev)] = dia_part 
+        std = self.get_config('radionuclide:particle_diameter_uncertainty')
+        if std > 0:
+            logging.debug('Adding uncertainty for particle diameter: %s m' % std)
+            self.elements.diameter[(sp_out==self.num_prev) & (sp_in!=self.num_prev)] += np.random.normal(
+                    0, std, sum((sp_out==self.num_prev) & (sp_in!=self.num_prev)))
+
+        # Transfer to slowly reversible particles
+        if self.get_config('radionuclide:slowly_fraction'):
+            self.elements.diameter[(sp_out==self.num_psrev) & (sp_in!=self.num_psrev)] = dia_part 
+            if std > 0:
+                logging.debug('Adding uncertainty for slowly rev particle diameter: %s m' % std)
+                self.elements.diameter[(sp_out==self.num_psrev) & (sp_in!=self.num_psrev)] += np.random.normal(
+                    0, std, sum((sp_out==self.num_psrev) & (sp_in!=self.num_psrev)))
+
+        # Transfer to irreversible particles
+        if self.get_config('radionuclide:irreversible_fraction'):
+            self.elements.diameter[(sp_out==self.num_pirrev) & (sp_in!=self.num_pirrev)] = dia_part 
+            if std > 0:
+                logging.debug('Adding uncertainty for irrev particle diameter: %s m' % std)
+                self.elements.diameter[(sp_out==self.num_pirrev) & (sp_in!=self.num_pirrev)] += np.random.normal(
+                    0, std, sum((sp_out==self.num_pirrev) & (sp_in!=self.num_pirrev)))
+ 
+        # Transfer to LMM
+        self.elements.diameter[(sp_out==self.num_lmm) & (sp_in!=self.num_lmm)] = dia_diss
+        
+        # Transfer to colloids
+        if self.get_config('radionuclide:species:Colloid'):
+            self.elements.diameter[(sp_out==self.num_col) & (sp_in!=self.num_col)] = dia_diss
+
+
+
+
+    def bottom_interaction(self,Zmin=none):
```

Comment:
This seems like the only place where `none` (lowercase) is used, was that intended? and if not then one import can also be removed.